src/Symfony/Component/Yaml/Inline.php

```diff
@@ -28,6 +28,7 @@ class Inline
     private static $exceptionOnInvalidType = false;
     private static $objectSupport = false;
     private static $objectForMap = false;
+    private static $constantSupport = false;
```

Comment:
Depends on whether we want to throw exceptions when a constant is not defined. Imo the flag makes sense if we then throw an exception when a constant does not exist (which would make debugging a lot easier than when we just returned `null`). Though if we enabled this feature unconditionally, we would not be able to throw exceptions when the constant was not defined.